### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by garyking**
Adding types.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for creating a TypeScript definition. 🙌 Can you follow [this styleguide](https://github.com/sindresorhus/typescript-definition-style-guide)?

Some examples:
- https://github.com/sindresorhus/semver-regex/commit/039944bb5b3876913171b895488874b5c200e458
- https://github.com/sindresorhus/global-dirs/commit/0aae4d64887f77af94c4fb7ca12aa9f93c0a8291
- https://github.com/sindresorhus/md5-hex/commit/bbbfeff840f26973fbffc16d4863ae6d120079f3

**Comment by sindresorhus** [comment]
This is already pretty close. Can you use `declare const` instead of a namespace? https://github.com/sindresorhus/typescript-definition-style-guide#documentation

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+declare namespace processExists {
+	/**
+	Check multiple processes if they exist.
+
+	@param input - Process IDs or names to check.
+	@returns - A map, with the process name/ID as key and the status as a boolean value.
+	*/
+	function all(
+		input: (number | string)[],
```
Comment (sindresorhus) [inline index.d.ts]: https://github.com/sindresorhus/typescript-definition-style-guide#prefer-read-only-values

**Comment by garyking** [comment]
@sindresorhus All done.

**Pull request merged**